css/properties/text-anchor.json

```
@@ -10,12 +10,14 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "1"
             },
             "chrome_android": "mirror",
-            "edge": "mirror",
+            "edge": {
+              "version_added": "≤15"
```

```suggestion
              "version_added": "≤14"
```

See: https://github.com/caugner/css-properties/blame/1609363b6575225215aadb935e3b22b59f26f2ce/properties.css#L282